backend/api/oauthEnabledClient.test.js

```diff
@@ -6,23 +6,24 @@ const logger = require('../log')
 const hostname = 'http://localhost:8080'
 
 describe('Test clients built by oauthEnabledClient', () => {
+  const client = new Client({ baseUrl: `${hostname}/`, timeout: 2000 })
+
+  afterEach(() => {
+    nock.abortPendingRequests()
```

Comment:
new version of node crashes if unhandled promises - this is required to clean up properly